skll/experiments/output.py

```diff
@@ -390,7 +398,7 @@ def _write_skll_folds(skll_fold_ids, skll_fold_ids_file):
     skll_fold_ids_file.flush()
 
 
-def _write_summary_file(result_json_paths, output_file, ablation=0):
+def _write_summary_file(result_json_paths: List[str], output_file: IO[str], ablation: int = 0):
```

Comment:
```suggestion
def _write_summary_file(result_json_paths: List[str], output_file: IO[str], ablation: Optional[int] = 0):
```

Reply:
`Optional` only indicates that a parameter can be `None`, not 0.